de.metas.contracts/src/main/java/de/metas/flatrate/api/impl/DefaultFlatrateHandler.java

```diff
@@ -98,4 +98,16 @@ public void afterExtendFlatrateTermCreated(I_C_Flatrate_Term oldTerm, I_C_Flatra
 	{
 		// nothing
 	}
+	
+	@Override
```

Comment:
i suggest to add the "does nothing" javadoc where to (and to beforeFlatrateTermCreated())